documentation/src/main/asciidoc/topics/ref_query_performance.adoc

```diff
@@ -0,0 +1,39 @@
+[id='query_performance-{context}']
+= Query Performance Tuning
+{brandname} exposes statistics for queries and provides configurable attributes so you can monitor and tune queries for optimal performance.
+
+.Check index usage statistics
+
+Indexing caches improves query performance.
+However, in some cases, queries might only partially use an index such as when not all fields in the schema are annotated.
+
+Start tuning query performance by checking the time it takes for each type of query to run.
+If indexed queries are not faster than non-indexed queries, you should make sure that caches are fully indexed by double checking entities and field mappings.
+
+.Index commit interval
+
+Indexing can degrade write throughput for {brandname} clusters.
+The `commit-interval` attribute defines the interval, in milliseconds, between which index changes that are buffered in memory are flushed to the index storage and a commit is performed.
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<index-writer commit-interval="2000" ...>
+  <index-merge .../>
+</index-writer>
+----
+
+This operation is costly so you should avoid configuring an interval that is too small. The default is 1000 ms (1 second).
+
+.Index refresh interval
+
+The `refresh-interval` attribute defines the interval, in milliseconds, between which {brandname} reads the index.
```

Comment:
The interval is for the refresh, not how often it reads the index. The index is always read on every query